<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkspacePlayer</class>
 <widget class="QMainWindow" name="WorkspacePlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WorkspacePlayer</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background-color:  transparent;
    color: #000000;
}

#topBar {
    border: 0;
	background-color: rgb(197, 192, 193);
}

#globalLocusBar, QToolButton {
    background-color: rgb(221, 217, 216);
    border: 0;
    border-radius: 9px;
}

#globalLocusBar:hover, QToolButton:hover {
    background-color: rgb(231, 227, 226);
}

QToolButton:pressed {
    background-color: rgb(211, 207, 206);
}

QStackedWidget {
  background-color: rgba(0, 0, 0, 0);
}

#centralwidget {
    background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 #CDC9C7, stop:1 #A0A9B4);
}

#spaceArea {
    border: 0;
}

SpaceCard, Dialog, BlockViewer, #flowListBox {
    border-radius: 9px;
	background-color: rgb(221, 217, 216);
}

SpaceCard:hover {
	background-color: rgb(231, 227, 226);
}

SpaceCard:pressed {
    border-radius: 18px;
	background-color: rgb(211, 207, 206);
}

#globalLocusBar, QLabel, SpaceCard &gt; QLabel {
    color: rgba(0, 0, 0, 180);
}

SpaceCard &gt; QToolButton, SpaceCard &gt; QPushButton{
    border: 0;
}

QLineEdit, TextInput {
   padding: 9;
   border-radius: 9px;
	background-color: rgba(255, 255, 255,70);
}

QPushButton {
	border: 0; 
	color: rgb(0,0,0);
	text-align: left;
	padding: 9;
	border-radius: 9px;
}

QPushButton:hover {
	background-color: rgba(52, 152, 219,70)
}

QPushButton:pressed {
	background-color: rgba(52, 152, 219,150)
}

QListView{
    outline: none;
}

QListView::item {
    border: 0;
	padding: 9;
	text-align: left;
	color: rgb(0,0,0);
	border-radius: 9px;
	background-color: transparent;
}

QListView::item:hover {
	background-color: rgba(52, 152, 219,70);
}

QListView::item:hover {
	background-color: rgba(52, 152, 219,70);
}

QListView::item:selected  {
	background-color: rgba(52, 152, 219,150);
}

Dialog &gt; QPushButton {
  background-color: rgba(52, 152, 219,70);
}

Dialog &gt; QPushButton:hover {
  background-color: rgba(52, 152, 219,128);
}

Dialog &gt; QPushButton:pressed {
  background-color: rgb(52, 152, 219);
}

#toggleListbox::checked {
  background-color: rgba(52, 152, 219,128);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="topBar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>9</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="welcome_message">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Hello, Natanael</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="internet">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>BlankCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="bluetooth">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>BlankCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="battery">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>BlankCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="volume">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>BlankCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="clock">
         <property name="text">
          <string>??:??</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="logout">
         <property name="minimumSize">
          <size>
           <width>37</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>37</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>BlankCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="shutdown">
         <property name="minimumSize">
          <size>
           <width>37</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>37</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>BlankCursor</cursorShape>
         </property>
         <property name="text">
          <string>Ã—</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="home">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QWidget" name="flowListBox" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>274</width>
          <height>242</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QPushButton" name="addFlow">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>â€€Add Flow...</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/symbolic/list-add.svg</normaloff>:/icons/symbolic/list-add.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="flowList">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="toggleListbox">
        <property name="geometry">
         <rect>
          <x>-80</x>
          <y>290</y>
          <width>80</width>
          <height>26</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 0;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
